src/Sign.Core/SignatureProviders/ClickOnceSignatureProvider.cs

```diff
@@ -78,116 +82,114 @@ public async Task SignAsync(IEnumerable<FileInfo> files, SignOptions options)
             using (X509Certificate2 certificate = await _certificateProvider.GetCertificateAsync())
             using (RSA rsaPrivateKey = await _signatureAlgorithmProvider.GetRsaAsync())
             {
-                // This outer loop is for a .clickonce file
+                // This outer loop is for a single deployment manifest (.application/vsto).
```

Comment:
Remove `single`.  Although the caller passes one now, a caller could pass several in the future, and this code handles both cases.